keyboards/arabica37/ssd1306.c

```diff
@@ -0,0 +1,344 @@
+#ifdef SSD1306OLED
```

Comment:
I would really prefer that the OLED Driver or QWIIC OLED feature be used here. 

Or worst case, that the /drivers/avr/ssd1306.c file be used instead, rather than having another copy of this sitting in a keyboard folder.  (this is not a knock against you, at all, but it's better if common code is used, since you don't miss out on fixes and improvements by doing so).